### PR TITLE
Added Instructions for step 1 & 2 and made the challenge code striaghtforward

### PR DESCRIPTION
**Pull request by xypnox**
I have added detailed instructions for steps 1 & 2 and a simple english explanation of what the bot does and how to make it work.

Moreover, I have moved the challenege condition above and put it in a conditional. It makes it easier to setup the app since less manual work is required. I had this problem in the beginning hence there are so many "verify *" commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,28 @@
...
+# How does it work?
+
+This bot watches for messages in the slackbot chat and whenever a message arrives with an attached email, It forwards it to a channel as a message.
+
+# How to make it work ?
 1. Set up a forwarding email. If you send any email to this address, it will appear as a file in your Direct Messaging channel with slackbot.
-1. Add this address as a forwarding address of your email account. (For gmail find forwarding options from gmail settings)
-1. Use Slack API to get notified about this.
-1. Use Slack API to post the email on a channel you want to.
+2. Add this address as a forwarding address of your email account. (For gmail find forwarding options from gmail settings)
```
Comment (OrkoHunter) [inline README.md]: It is actually a good idea to use `1.` for all the list items. Adding or removing items gets easier without making us change all the indices.

Comment (xypnox) [reply README.md]: Hmm I never knew this syntax. Updating code :+1:

**Comment by OrkoHunter** [comment]
Great! Merge it.

**Comment by xypnox** [comment]
> Great! Merge it.

Can't. Not a collaborator :cry:

**Comment by OrkoHunter** [comment]
Added the team!

**Pull request merged**